### PR TITLE
Add jvz and danielbeck to sshd module, fix file name, remove KK

### PR DESCRIPTION
**Pull request by daniel-beck**
https://github.com/jenkinsci/sshd-module

@oleg-nenashev @jvz 

### Always

- [x] Add link to plugin/component Git repository in description above

### For a newly hosted plugin only

n/a

### For a new permissions file only

n/a

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@daniel-beck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by slide** [comment]
We generally request a sign-off from anyone being removed. I am not sure if KK has already said he can be removed from stuff because he is no longer heavily involved with Jenkins. I defer to others who may know more about that.

**Comment by daniel-beck** [comment]
Probably something you should check with KK to potentially let us clean up these files and status of some components. I am fairly sure he told me long ago he doesn't consider himself a maintainer of anything anymore, but I cannot find a reference.

**Review by oleg-nenashev — APPROVED** [review]
According to my interpretation of the process, any Jenkins core maintainer is eligible to get access to all Jenkins core modules and libraries, without additional approval. No problem to remove KK, he explicitly stepped down from the Jenkins maintenance a while ago. Access can be restored later if needed.

**Comment by oleg-nenashev** [comment]
I cannot find a reference as well, but I will go ahead to get others the required permissions. If @kohsuke would like to get permissions to this module due to whatever reason, I will restore access in a follow-up pull request

**Pull request merged**
